tools/c7n_mailer/README.md

```diff
@@ -516,6 +516,16 @@ The mailer will transmit all messages found on the queue on each execution, and
 sending 3 times in the event of a failure calling SendGrid.  After the retries the queue
 message will be discarded.
 
+In addition, SendGrid delivery on Azure supports using resource tags to send emails. For example, in the `to` field:
```

Comment:
@logachev,  I believe this tag reference feature should work with other transports other than SendGrid, we should just make these docs generic if that is the case.

Reply:
agreed. It also works with SMTP delivery (and not just Azure, AWS too)

Reply:
I think it's Azure specific behavior. AWS supports a `contact_tags` field in the `mailer.yml` file where a list of tags are stored and each resource is checked for these tags. If the resource has the tag, it adds the value of the tag to target.
<img width="808" alt="Screen Shot 2019-10-07 at 10 53 03 AM" src="https://user-images.githubusercontent.com/19538721/66335859-b8b25200-e8f0-11e9-8eb0-8eca76f8f426.png">

I wouldn't want to make it generic docs if it's specific to Azure